### ISSUE TITLE
 Invalid 'where' argument: 'accounts-password'

### ISSUE THREAD

**Issue by conrad-dk** [etyp/meteor-accounts-ldap#28]
This happens when cloning the project to /packages, not when installing via atmosphere. It happens regardless of whether packages accounts-password and accounts-base are installed. 

I suspect this is my own fault somehow, but if so it might be worth clarifying in the instructions.

**Comment by chazsolo** [comment]
Did you find a solution to this?

**Comment by chazsolo** [comment]
Figured this was part of the `api.imply` call, changing it to 

```
api.imply(['accounts-base', 'accounts-password'], ['client', 'server']);
```

instead of

```
api.imply('accounts-base', 'accounts-password', ['client', 'server']);
```

fixed that for me.

**Comment by conrad-dk** [comment]
Fixed in ecb08b4.
